apps/fabric-website/src/components/Site/Site.tsx

```diff
@@ -191,16 +192,25 @@ export class Site<TPlatforms extends string = string> extends React.Component<IS
   }
 
   private _renderMessageBar(): JSX.Element | null {
-    const { pagePath } = this.state;
+    const { pagePath = '' } = this.state;
     // TODO: generalize when to show a message bar and how to provide the text
-    if (pagePath && pagePath.indexOf('#/controls/web') === 0 && pagePath.indexOf('fluent-theme') === -1) {
+    if (pagePath && pagePath.indexOf('#/controls/web/') === 0 && pagePath.indexOf('fluent-theme') === -1) {
```

Comment:
This is super hacky, but it works for now--definitely plan on updating this to a better approach post-//build